centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -121,12 +122,41 @@ object Operations extends StrictLogging {
     storageOptions.getService
   }
 
+  implicit private val timer = IO.timer(global)
+  implicit private val contextShift = IO.contextShift(global)
+
   def submitWorkflow(workflow: Workflow): Test[SubmittedWorkflow] = {
     new Test[SubmittedWorkflow] {
       override def run: IO[SubmittedWorkflow] = CentaurCromwellClient.submit(workflow)
     }
   }
 
+  def checkDescription(workflow: Workflow, validityExpectation: Option[Boolean]): Test[Unit] = {
+    new Test[Unit] {
+      override def run: IO[Unit] = {
+        // We can't describe workflows based on zipped imports, so don't try:
+        if (workflow.skipDescribeEndpointValidation || workflow.data.zippedImports.nonEmpty) {
```

Comment:
log this or would that be too much?

Reply:
Log that we aren't testing it because it uses zip imports? I guess my question is, who would really use that log output?

Reply:
I'm not sure, hence my hesitation. But presumably the addition of "describe" in most flows is adding value that wouldn't be added here.

Reply:
Added a message at "warn" level.

Reply:
Will logger.warn end up sending tons of warnings to Sentry?

Reply:
not from centaur testing, AFAIK

Reply:
LGTM